### DISCUSSION TITLE
Renovate treats Android ABI splits as dependencies

### DISCUSSION THREAD

**Discussion by FooIbar** [renovatebot/renovate#28954] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Renovate gives the following warning:
```
Renovate failed to look up the following dependencies: Failed to look up maven package arm64-v8a:x86_64.
```
Here is the content in the affected file:
```
android {
    splits {
        abi {
            isEnable = true
            reset()
            include("arm64-v8a", "x86_64", "armeabi-v7a")
        }
    }
}
```

**Comment by FooIbar** [comment]
https://github.com/FooIbar/renovate-abi-repro
Here is a minimal repro.

**Reply by rarkins** [reply]
@Churro do you think this is a pattern which could be ignored? I searched for the string "include(" in our repo expecting to find some Gradle fixtures or tests but didn't find any

**Reply by Churro** [reply]
I think it's an edge case which could more easily be handled by a package rule that disables updates for the false dependency `arm64-v8a:x86_64:armeabi-v7a`. 

renovate already had a list of methods to ignore, which was then replaced by a more generic approach (https://github.com/renovatebot/renovate/pull/21753). The reason this doesn't work here is because `armeabi-v7a` includes an integer (checked [here](https://github.com/zharinov/renovate/blob/46272fbd3ec3514f2578a08a1cd80df7e8732a4f/lib/modules/manager/gradle/utils.ts#L27)), which is "unfortunately" enough to consider it a valid dependency.

**Comment by FooIbar** [comment]
We ended up using `ignorePaths` as the current rules are too permissive, resulting in false positive on something like
```
buildConfigField("long", "BUILD_TIME", "0")
```
as well.
